src/main/java/com/atherys/towns/facade/PlotFacade.java

```diff
@@ -111,6 +112,14 @@ public TownPlot getPlotAtPlayer(Player player) throws TownsCommandException {
                 new TownsCommandException("You are not standing in a plot."));
     }
 
+    public boolean checkHostileSpawningAtLocation(Location<World> location) {
+        AtomicBoolean spawn = new AtomicBoolean(true);
```

Comment:
You don't need to use an atomic here. Just ` return .map(plot -> plot.getTown().isMobs).orElse(false)`.